cloudbuild/unit_test_jobs.jsonnet

```diff
@@ -34,7 +34,7 @@ local unittest = base.BaseTest {
     )
   ],
   command: [
-    'pytest --run_large --durations 0',
+    'pytest --run_extra_large --durations 0',
```

Comment:
let's revert this before submit. also, are there any large test that aren't getting picked up here?

should we try `pytest --run_extra_large --durations 0 keras_cv/`? or have we run that manually?

Reply:
Let's keep this going since the runtime is 12 mins for JAX gcb testing, not bad!